### PR TITLE
OKTA-549183 : Update Smart Card icons

### PR DESCRIPTION
**Pull request by leemchale-okta**
## Description:

This PR updates the Smart Card AuthCoin icon and button icon.  It also adds support for button icons.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [x] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-549183](https://oktainc.atlassian.net/browse/OKTA-549183)

### Screenshot/Video:

image
image

### CODE REVIEW

File: src/v3/src/img/smartCardButtonIcon.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+  <path fill="#1D1D21" fill-rule="evenodd" d="M6 1h13a1.5 1.5 0 0 1 1.5 1.5v19A1.5 1.5 0 0 1 19 23H6a1.5 1.5 0 0 1-1.5-1.5v-19A1.5 1.5 0 0 1 6 1ZM3.5 2.5A2.5 2.5 0 0 1 6 0h13a2.5 2.5 0 0 1 2.5 2.5v19A2.5 2.5 0 0 1 19 24H6a2.5 2.5 0 0 1-2.5-2.5v-19ZM11.502 15h-1v1h1v-1Zm-3 0h1v1h-1v-1Zm0 5v-3h3v1h-2v2h-1Zm2-1h1v1h-1v-1Zm-2-5a1 1 0 0 0-1 1v5a1 1 0 0 0 1 1h3a1 1 0 0 0 1-1v-5a1 1 0 0 0-1-1h-3Z" clip-rule="evenodd"/>
```
Comment (glenfannin-okta) [inline src/v3/src/img/smartCardButtonIcon.svg:2]: Should this be `fill="currentColor"` to inherit the color? Just curious

Comment (leemchale-okta) [reply src/v3/src/img/smartCardButtonIcon.svg:2]: good point I think it does so the icon changes color when the user hovers over the button? @alexdahl-okta

File: src/v3/src/components/Button/Button.tsx [hunk 2]
```diff
@@ -64,6 +65,42 @@ const Button: UISchemaElementComponent<{
...
+        // TODO: OKTA-518793 - replace english string with key once created
+        aria-label="Loading..."
+        aria-valuetext="Loading..."
+        sx={{ color: 'white' }}
+      />
+      {label}
+    </Box>
+  );
+
+  const buttonLabel = Icon ? (
```
Comment (glenfannin-okta) [inline src/v3/src/components/Button/Button.tsx:88]:
```suggestion
  const buttonLabel = typeof Icon !== 'undefined' ? (
```

**Review by glenfannin-okta — APPROVED** [review]

File: src/v3/src/img/smartCardButtonIcon.svg (hunk 1 above)
Comment (alexdahl-okta) [reply src/v3/src/img/smartCardButtonIcon.svg]: Sorry I was late here! That's cool, good idea I hadn't thought of. Should be good!

**Pull request merged**